### ISSUE TITLE
UI: Change Font Size of Social Links

### ISSUE THREAD

**Issue by Sriparno08** [Sriparno08/Openpedia#221]
## Category

- [ ] Documentation
- [ ] Resource Addition
- [ ] Codebase
- [x] User Interface
- [ ] Feature Request

## Description

We need to change the font size of the social links in the landing page footer to `23px`:

<p align="center">
  <img src="https://github.com/Sriparno08/Openpedia/assets/89148144/5ce0e874-6ce8-4fb3-ab92-ba664f81959d">
</p>

**For Assignee:**

> Ensure that you submit a PR for this issue within 1 day. Otherwise, you will be unassigned from this issue and it will be passed on to the next contributor who asked to be assigned to it.

_Note: This issue is for JWoC 2024 participants. If you are not participating in this event, please refrain from asking to get assigned to this issue._

**Comment by Hemant5104** [comment]
i would like to work on this issue under jwoc'24

**Comment by vishal2005025** [comment]
I have read the issue and I would like to work on this issue.
Please assign it to me under JWOC'2024.

**Comment by Sriparno08** [comment]
@SharmaDhruv2511 & @Hemant5104, please mention the 'JWOC' (Case-Sensitive) keyword in your comment to be considered for assignment.

The first one to do it will be assigned.

@vishal2005025 You're already assigned to another issue. Once your PR for it is merged, you will be considered for other issues.

**Comment by Hemant5104** [comment]
I would like to work on that issue under JWOC'24

**Comment by Sriparno08** [comment]
@Hemant5104 You're assigned.

**Comment by Hemant5104** [comment]
Please review my PR #239

**Comment by vermaanurag1532** [comment]
please Assign it to me

**Comment by vikki9121** [comment]
Please assign to me
